logstash-core-event-java/src/main/java/com/logstash/Javafier.java

```diff
@@ -49,6 +81,20 @@ public void visit(IRubyObject key, IRubyObject value) {
         return result;
     }
 
+    private static Object deepAnything(Object o) {
```

Comment:
why don't we have this be `deep(Object o)`? and separate the concerns accordingly as we get more specific